custom_json_diff/custom_diff.py

```diff
@@ -19,16 +19,16 @@ def calculate_pcts(diff_stats: Dict, j1: BomDicts, j2: BomDicts) -> List[list[st
     j1_counts = j1.generate_counts()
     j2_counts = j2.generate_counts()
     result = [
-        [f"Common {key} matched: ", f"{value}"]
-        for key, value in diff_stats["common"].items()
+        [f"Common {key} matched: ", f"{value}/{j1_counts[key] + j2_counts[key]} ({round((value / (j1_counts[key] + j2_counts[key])) * 100, 2)}%)"]
```

Comment:
@prabhu The math isn't correct because common components are only counted once even though they are contained in both.

Reply:
I think this is fine. Since they are common we shouldn't double count otherwise the percentage will be much lower

Reply:
@prabhu No, it isn't right because this way, you're counting all the components twice EXCEPT in the numerator, completely skewing percentages in a negative direction. Even if every single thing matched, you'd never get to 100%.

Example
bom A has 50 components
bom B has 50 components
We find 50 common components because everything matches
By this math, you'd be saying that only 50% of components matched because 50/(bom A + bom B) = 0.5
So you can't count elements from both boms but not double the common components

I have updated and I do not show the doubled number, but I did double the number for the purposes of percentages.